src/Surfnet/StepupMiddleware/ApiBundle/Request/CommandValueResolver.php

```diff
@@ -43,12 +43,17 @@ public function resolve(Request $request, ArgumentMetadata $argument): iterable
 
         $this->assertIsValidCommandStructure($data);
 
-        $commandName = [];
-        preg_match('~^(\w+):([\w\\.]+)$~', (string)$data['command']['name'], $commandName);
+        $commandNameParts = [];
+        preg_match('~^(\w+):([\w\\.]+)$~', (string)$data['command']['name'], $commandNameParts);
+        if (count($commandNameParts) < 1) {
```

Comment:
Should that not be `< 2` or `< 3` since on lince 55/56 indexes 1 and 2 are used?

Reply:
But the regex will make sure it's >1 or 0.
I shall add a test and use the result of the `preg_match` to hopefully make it more readable.

Reply:
ah, yeah. Because there are 2 captures :+1: